universal-application-tool-0.0.1/app/services/question/QuestionServiceImpl.java

```diff
@@ -81,13 +81,12 @@ public boolean addTranslation(
     // it is applied directly.  To a deleted question, it is ignored.  To an active or
     // obsolete question, to the draft for that question.
     switch (question.getLifecycleStage()) {
-      case DRAFT:
-        question = questionRepository.updateQuestionSync(new Question(definition));
-        break;
       case DELETED:
         return ErrorAnd.error(
             ImmutableSet.of(
                 CiviFormError.of(String.format("Question %d was DELETED.", definition.getId()))));
+      case DRAFT:
```

Comment:
(non-blocking): at this point should it be `if(DELETED) throw`, else do the normal thing?

Reply:
Done.